open-data-day-tweets-2018/README.md

```diff
@@ -0,0 +1,13 @@
+
+# About the Open Data Day Tweets Data Package
```

Comment:
Follow the pattern set on https://frictionlessdata.io/guides/publish-faq/. The sections here would be:

```
# OpenDataDay 2018 Tweets

[The 7th International Open Data Day](http://opendataday.org) was marked on March 3, 2018. On the day, people used the [#opendataday](https://twitter.com/hashtag/OpenDataDay) and [#odd18](https://twitter.com/hashtag/ODD18) hashtags to share insights from over 400 community events.

## Data

The data package contains ...

## Preparation

To run the scripts you need R version blah blah. If that's installed, run it via `Rscript blah blah`.

## License

The data is by twitter, blah blah, everything else, including scripts, blah blah is under the CC-0 Public Domain Waiver.
```

Note that it's slightly different from the FAQ link, where I added the name of the dataset in the first header.

Reply:
I think adding the image in the README is cool too, like https://github.com/vitorbaptista/birmingham_schools. However, I'm not sure if we're able to do legally share this map at all.